### DISCUSSION TITLE
What does an XHR status code of 0 actually mean?

### DISCUSSION THREAD

**Discussion by JeremyBradshaw7** [getsentry/sentry-react-native#2540] [Q&A]
We're getting many many instances like this in Sentry for our requests that we initiate through Axios:

![image](https://user-images.githubusercontent.com/36155348/195290470-f02a31ab-2152-4e84-beb9-8ef0c2fed759.png)

What does the status code of [0] actually indicate? Is it a network/comms issue with the device?

**Comment by marandaneto** [comment]
It's the `status_code`, but it was likely not available.

**Reply by pke** [reply]
That's usually a network issue. Most likely a connection drop, connectivity problems or bad SSL pinning.
The underlying fetch implementation can't connect to the server so it defaults to HTTP status code 0.

**Reply by marandaneto** [reply]
@krystofwoldrich maybe raise an issue on the JS repo to ignore the response code in case its not available?

**Reply by pke** [reply]
This could point to a serious connection problem the app is facing, I wouldn't ignore this code.
